examples/docker/shiny-core/app-core.py

```diff
@@ -0,0 +1,151 @@
+from __future__ import annotations
```

Comment:
this is only needed for Python 2, I don't think it's used in this script

Reply:
[Removed](https://github.com/ploomber/doc/pull/105/files#diff-c99010220192b232032cc417d9e26e799aa65a2dd2fbb5dd79b5595548293670R1-R3) from `app.py`